### DISCUSSION TITLE
Consiglio acquisto termovalvola smart essenziale

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#2146] [Hardware & Firmware]
La mia casa è già dotata di termostati Tado e di alcune loro termovalvole.
Su di un radiatore ho necessità di montare una termovalvola comandabile da Home Assistent ma non in termini di temperatura bensì di apertura della stessa come se fosse una termovalvola manuale. Non voglio impostare la temperatura ma l'apertura da 0 a 5 fregandomene della temperatura.
Cerco pertanto una termovalvola economica con tale funzionalità e connettività wifi da comandare localmente da Home Assistant. 

Mi sapreste consigliare qualche prodotto? In rete trovo solo roba basata sulla temperatura e spesso utilizzano ZigBee.

**Comment by indomus** [comment]
> La mia casa è già dotata di termostati Tado e di alcune loro termovalvole. Su di un radiatore ho necessità di montare una termovalvola comandabile da Home Assistent ma non in termini di temperatura bensì di apertura della stessa come se fosse una termovalvola manuale. Non voglio impostare la temperatura ma l'apertura da 0 a 5 fregandomene della temperatura. Cerco pertanto una termovalvola economica con tale funzionalità e connettività wifi da comandare localmente da Home Assistant.

difficilmente la troverai che funzioni in quel modo, ed è normale che sia così. apertura 0-5 purtroppo non significa nulla, proprio perché le valvole sul termosifone sulle quali vai a montare la termovalvola hanno escursioni differenti da modello a modello - non a caso, quelle intelligenti come primo step di installazione fanno la calibrazione, dopodiché le controlli con la temperatura.

**Reply by snakuzzo** [reply]
Se ho inteso bene cosa vuoi fare, puoi farlo pure con le aTado che hai...giocando sull'offset della temperatura per tenerla sempre aperta.

Se la imposti ad OFF...rimane ad OFF
Per tenerla sempre aperta puoi sfruttare le API di Tado ed impostare con uno script o un'automazione...

- offset temperatura a -9ºC (che è il limite)
- temperatura target a 25ºC

In questo modo la testa non chiuderà mai la valvola ... A meno che tu non sia in un ambiente con temperatura più alta di 34ºC :D

Io faccio così con le Tado per una sorta di BOOST. Fai sapere se ti occorrono info...nel caso ti faccio un esempio

**Reply by array81** [reply]
Si al limite potrei fare così, è che non mi andava di "sprecarci" una Tado visto i costo e l'uso ON/OFF che mi occorre.
Il termosifone in questione si trova in un garage che non deve essere riscaldato. Avendo una stufa a pellet quando tutti i termostati di piano si chiudono (e di conseguenza le valvole di zona), può capitare (raramente ma capita) che nel mentre che la stufa si spenga la temperatura dell'acqua aumenti troppo non avendo "sfogo" a quel punto la stufa si spegne ma si attiva una procedura di standby che prevede che non si riaccenda fin tanto che la temperatura dell'acqua non scende di 20 gradi anche se nel frattempo i termostati si accendono.
Lasciando la valvola del termosifone in garage aperta, dato che la linea del garage è diretta (non ha valvole di zona) la cosa non succede mai ma si spreca inutilmente del calore.
La mia idea era di mettere una valvola smart (ora è manuale sempre aperta) e programmarla tramite HA in modo che quando almeno uno dei termostati è acceso la valvola si chiuda (cosi da non sprecare calore visto che il garage è freddo) mentre quando tutti i termostati sono spenti si apra in quanto la stufa si sta spengendo ed è bene che il calore residuo venga dissipato.
A me quindi basterebbe qualcosa del tipo aperto/chiuso. Tutta la parte smart delle TADO non sarebbe sfruttata.
